public/js/events.js

```diff
@@ -3,7 +3,8 @@ const handleLocationChange = async (value) => {
   const newUrl = `${baseUrl}${countiesLocation[value].zoom}/${countiesLocation[value].latitude}/${countiesLocation[value].longitude}`;
 
   const county = countiesLocation[value].name;
-  const numberOfHouseholds =await getHouseHoldCount(county);
+  countyGraphChange(county);
```

Comment:
Where would this get data from?

Reply:
Refactored the countyGraphChange method to have access to the data @kilemensi

Reply:
I don't see how

Reply:
Made a change @kilemensi take a look